packages/puppeteer/package.json

```diff
@@ -34,7 +34,7 @@
     "express": "^4.16.4",
     "mocha": "^5.2.0",
     "nyc": "^14.1.1",
-    "puppeteer": "^1.10.0",
+    "puppeteer": "latest",
```

Comment:
Since there is a `package-lock.json` file here, using `latest` doesn't really mean "latest". I think we ought to pin v5 here instead.